drift/info_parser.py

```diff
@@ -0,0 +1,43 @@
+DRIFT_FACTS = {'bios_uuid', 'display_name'}
```

Comment:
Do we want to care about 'trimming' the list of facts vs. showing all available facts?

Reply:
When the data comes back, it's mostly various IDs that wouldn't make sense to compare. There's a `facts` list as well but it's empty currently.

I think eventually we'll just be using the `facts` list, but for now I use this set to create some ersatz facts based on the most easily recognized IDs :blowfish: